src/Routers/FilesRouter.js

```diff
@@ -94,6 +94,29 @@ export class FilesRouter {
 
   async createHandler(req, res, next) {
     const config = req.config;
+    if (
+      !req.config.fileUpload.enabledForAnonymousUser &&
+      req.auth.user &&
+      Parse.AnonymousUtils.isLinked(req.auth.user)
```

Comment:
This would deny file upload for anonymous users by default, i.e. if the configuration parameter is not set. Did we already discuss what we want as default values for all the file upload settings?